orchestrator/gravity_utils/src/prices.rs

```diff
@@ -109,3 +185,36 @@ pub async fn get_dai_price(
     web3.get_uniswap_v2_price(pubkey, token, *DAI_CONTRACT_ADDRESS, amount, None)
         .await
 }
+
+#[cfg(test)]
+mod tests {
+    use actix::System;
+    use clarity::Address;
+    use clarity::Uint256;
+    use env_logger;
+    use std::time::Duration;
+    use web30::client::Web3;
+
+    use super::get_weth_price_with_retries;
+
```

Comment:
This test is a convenient way to fetch the cheqd token prices using the modified price fetching method.